source/release-notes.txt

```diff
@@ -10,6 +10,26 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.39.4
+----------------
+
+*Released September 6, 2023*
+
+Bug Fixes:
+
+- Allows ``[object Object]`` as a valid string value in TypeChecker 
+  (:issue:`COMPASS-7132`).
+- Doesn't treat non-numbers in CSV headers as array indexes 
+  (:issue:`COMPASS-7157`).
+- Limits when custom paste handling is applied; uses clipboard data when 
+  auto-fixing user input (:issue:`COMPASS-7149`).
+- Updates Electron to v24.8.2 to address security vulnerabilities. This updates 
+  specifically addresses CVE-2023-4427 and CWE-119.
+
+`Full Changelog available on GitHub
+<https://github.com/mongodb-js/compass/compare/v1.39.3...v1.39.4-beta.2>`__
```

Comment:
[question]

Is there a reason we're comparing with the beta branch? Can we use this URL instead?

https://github.com/mongodb-js/compass/compare/v1.39.3...v1.39.4

Reply:
Ah! That was my mistake — when I created the PR, the 1.39.4 release wasn't officially out yet. Updating.